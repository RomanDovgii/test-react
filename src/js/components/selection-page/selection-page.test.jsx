import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import SelectionPage from "./selection-page";

it(
    `Should render SelectionPage`,
    () => {
      const renderer = new ShallowRenderer();

      const tree = renderer
      .render(
          <SelectionPage/>
      );

      expect(tree).toMatchSnapshot();
    }
);
