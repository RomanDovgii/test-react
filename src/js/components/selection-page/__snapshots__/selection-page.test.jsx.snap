// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render SelectionPage 1`] = `
<section
  className="selection"
>
  <div
    className="selection__wrapper"
  >
    <p
      className="selection__text"
    >
      Выберите объем данных
    </p>
    <Link
      className="selection__link"
      to={
        Object {
          "pathname": "/table",
          "state": Object {
            "dataSize": "LARGE",
          },
        }
      }
    >
       Большой объем данных 
    </Link>
    <Link
      className="selection__link"
      to={
        Object {
          "pathname": "/table",
          "state": Object {
            "dataSize": "SMALL",
          },
        }
      }
    >
       Маленький объем данных
    </Link>
  </div>
</section>
`;
